src/rules/unit-blacklist/README.md

```diff
@@ -0,0 +1,44 @@
+# unit-blacklist
+
+Specify a blacklist of disallowed units.
+
+```css
+    a { width: 100px; }
+/**               ↑
+ *      These units */
+```
+
+## Options
+
+`array`: `"["array", "of", "units"]"`
+
+Given:
+
+```js
+["px", "em"]
+```
+
+The following patterns are considered warnings:
+
+```css
+a { width: 100px; }
+```
+
+```css
+a { font-size: 10em; }
+```
+
+The following patterns are *not* considered warnings:
+
```

Comment:
Can you throw in a `rem` example --- since I imagine the real-world use-case for blacklisting `px` and `em` is to enforce the use of `rem`.